### ISSUE TITLE
All sessions are lost, DB is empty, local backups were enabled but no sessions were saved

### ISSUE THREAD

**Issue by mrdc** [sienori/Tab-Session-Manager#596]
Hello,

That's a disaster! All sessions are lost. Local Backup was enabled, but I haven't noticed that backups weren't created.
![89954689-3eeaa380-dc3a-11ea-9548-eacc225965ba](https://user-images.githubusercontent.com/16876990/90002124-ebf50880-dc9a-11ea-8b62-7d003139de40.png)

Folder 1782160246ssensosi.files is empty.
1782160246ssensosi.sqlite is 30kb... so evething is gone. 
TSM 6.1.0, FF 79 32bit, W7 32bit.

It's the third or fourth time I'm loosing all sessions when db is big with lots of sessions.

**Comment by mrdc** [comment]
@sienori 
UPD: I've replicated the bug one more time:
I've imported 257 sessions from 2 TSM json files: 61 session + 196.
Then I've manually save ~ 80 tabs + 20 tabs. Closed FF79 (it's in private mode), reopened and all sessions are gone, sqlite db is empty, 1782160246ssensosi.files also empty. Local backup at start wasnt' made, but it's enabled in options.

**Comment by sienori** [comment]
Local backups are saved in the firefox download folder. e.g. `Downloads/TabSessionManager - Backup/`
Have you checked?

**Comment by mrdc** [comment]
> Local backups are saved in the firefox download folder. e.g. `Downloads/TabSessionManager - Backup/`
> Have you checked?

No, they weren’t saved before the DB crash (probably because it was too large). After the crash I see backups appearing in the backup folder.

**Comment by mrdc** [comment]
@sienori 
The same bug on FF80 x64, Win10 1909 x64. Sessions are not auto-saved.
![image](https://user-images.githubusercontent.com/16876990/91969561-f4cb7e00-ed1e-11ea-8f23-870ff0e19cce.png)
![image](https://user-images.githubusercontent.com/16876990/91969675-27757680-ed1f-11ea-844f-413b6f77a501.png)
![image](https://user-images.githubusercontent.com/16876990/91969703-32300b80-ed1f-11ea-9f0a-46fb9f0d8f9c.png)
